src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Loading.java

```diff
@@ -0,0 +1,77 @@
+package com.wikia.webdriver.testcases.globalnavigationbar;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.HomePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.globalnav.GlobalNavigationPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialWikiActivityPageObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Test(groups = {"globalnavigationbar", "Loading"})
+public class Loading extends NewTestTemplate{
```

Comment:
"Loading" class name is very non-intuitive for me. Could you rename to something like "Interactions" or "presence" at least? Although I can't say those names describes what happens inside those classes as well